waterlog/src/Components/WastageGraphComponent.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import DailyWastageComponent from './DailyWastage';
+class WastageGraphComponent extends Component {
+    constructor(props){
+      super(props);
+
+      this.openGraph = this.openGraph.bind(this);
+      this.state = {
+        display: "daily"
+      }
+    }
+    componentDidMount(){
```

Comment:
empty line between declarations

Reply:
Just deleted it because i wasn't using that component.